pkg/kubelet/cloudresource/cloud_request_manager_test.go

```diff
@@ -53,41 +36,41 @@ func createNodeInternalIPAddress(address string) []v1.NodeAddress {
 	}
 }
 
-func TestNodeAddressesRequest(t *testing.T) {
-	syncPeriod := 300 * time.Millisecond
-	maxRetry := 5
+func TestNodeAddressesDelay(t *testing.T) {
```

Comment:
This test is flaky on master. I'm wondering if we should remove the concurrency altogether.

Reply:
`sync.Cond` is tricky to reason about so I strongly recommend having a good concurrent test.

Reply:
Done.